src/flight_paths.py

```diff
@@ -0,0 +1,64 @@
+import requests
+from graph import Graph
+from distance import calculate_distance
+
+
+def import_cities():
+    '''
+        Import the city flight connection path JSON file,
+        Returns dictionary of cities
+    '''
+    # url = '///cities_with_airports.json'
+    url = 'https://codefellows.github.io/sea-python-401d4/_downloads/cities_with_airports.json'
+    r = requests.get(url)
+    cities = r.json()   # converts json to dictionary
+    return cities
+
+
+def populate_city_dict(cities):
+    '''Populates a weighted graph with city and conection info'''
+    d = {}
+    for index, city in enumerate(cities):
+        print('index: {} airport: {} lat_lon: {}'
```

Comment:
This will quickly clutter up your terminal. You don't really need this for production code.